target/linux/ramips/rt305x/base-files/etc/board.d/01_leds

```diff
@@ -66,6 +66,12 @@ asiarf,awapn2403)
 dlink,dcs-930l-b1)
 	ucidef_set_led_netdev "wifi" "WiFi" "$boardname:blue:wps"
 	;;
+dlink,dir-506l)
+	ucidef_set_led_gpio "lo_bat" "Low battery" "${boardname}:red:status" "10" "0"
+	ucidef_set_led_default "status" "Status" "${boardname}:green:status" "1"
+	ucidef_set_led_netdev "lan" "LAN" "${boardname}:green:usb" "eth0" "tx rx"
+	ucidef_set_led_netdev "wifi" "WiFi" "rt2800soc-phy0::radio" "wlan0" "tx rx"
```

Comment:
No "link"?

Reply:
I'm trying to avoid make cristmass tree  form device, it already have one green led that always on after boot.
Also this saves few mWatts battery power.